### DISCUSSION TITLE
OpenDTU in Gast Wlan bei Fritzbox

### DISCUSSION THREAD

**Discussion by H83v** [tbnobody/OpenDTU#1549] [General]
Hi zusammen,

erstmal Danke für all die Arbeit die hier von euch geleistet wird.

Ich bin ziemlicher Laie und brauche mal bei folgendem Thema Hilfe:
- ich habe die DTU über mein WLAN eingerichtet. Läuft alles soweit wunderbar.
Gerne würde ich aber die DTU über mein "Gäste Wlan" bei meiner Fritzbox laufen lassen, statt über mein "Standard Wlan" der Fritzbox. Macht das grundsätzlich Sinn oder seht ihr hier kein Problem, im "Standard Wlan" meines Routers mit der DTU zu bleiben?
- Wenn das mit dem Gäste Wlan Sinn macht: Wie kann ich dann am besten auf das Webinterface der DTU zugreifen, wenn ich mit meinem PC im "Standard Wlan" drin bin. Muss ich dann bei meinem Rechner immer erst auf das "Gäste Wlan" wechseln um das Webinterface ansteuern zu können?

Ich hoffe meine Fragen sind verständlich :-).

Grüße Hannes

**Comment by Tiese** [comment]
Hallo Hannes,
genau dafür ist das Gäste-WLAN da - damit man SEINE Zugangsdaten NICHT preisgibt!
Das ist im umgekehrten Fall natürlich genauso.

Schöne Grüße
Tiese

**Reply by H83v** [reply]
Ok, danke dir. Sprich ich muss mich dann immer von meinem "Standard Wlan" auf mein "Gäste Wlan" wechseln um auf das DTU Webinterface zugreifen zu können. Oder gibt es da technisch eine einfachere Lösung, trotz das die DTU im "Gäste Wlan" drin ist?

**Reply by Tiese** [reply]
Ich verstehe nicht so ganz, warum du sie im Gäste-WLAN haben willst &#129300;

**Reply by H83v** [reply]
Entschuldige die naiven Fragen. Da das OpenDTU Projekt ja ein Opensource Projekt ist und es nicht darum geht Daten zu sammeln, wird es sicherlich unbedenklich zu sein. Ich hatte vorher einen Deye Wechselrichter, bei dem ich nicht so recht wusste, wie er mit den Daten umgeht.

Daher hab ich die Frage gestellt, ob die DTU lieber ins Gäste Wlan gehört?

**Reply by Tiese** [reply]
Die Idee ist ja gut, aber deine Logik nicht.

Angenommen da ist Schadsoftware programmiert...
Du logst dich ins WLAN ein "EGAL" welches - ob Gast oder nicht - ist DEIN PC damit verbunden.
Wenn also die Software agiert, spielt es keine Rolle wie sich der Zugang nennt!

**Reply by H83v** [reply]
Ok, versteht. Danke für die Erläuterung.
